micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/MultiSelect.js

```
@@ -130,7 +152,18 @@
       setIsSeeAll(true);
     }
     setSelectedDetails(mainData?.filter((obj) => (multiple ? value?.includes(obj.uniqueIdentifier) : obj.uniqueIdentifier == value)));
-  }, [formattedOptions, optionsLimit]);
+    // Update formattedOptions2
+    let newFormattedOptions2 = [...formattedOptions];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newFormattedOptions2' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)